audbackend/core/artifactory.py

```diff
@@ -53,9 +54,12 @@ def _exists(
     ) -> bool:
         r"""Check if file exists on backend."""
         try:
-            # Can lead to RuntimeError: 404 page not found
+            # Can lead to
+            # RuntimeError: 404 page not found
+            # or
+            # requests.exceptions.HTTPError: 403 Client Error
             return audfactory.path(path).exists()
-        except RuntimeError:  # pragma: nocover
+        except self._non_existing_path_error:  # pragma: nocover
```

Comment:
Could it make sense to catch any `Exception` here to avoid the code breaks again in the future?

Reply:
In general it is not a good idea to catch simply everything, because you might also get an exception because you use your keyboard to interrupt or you don't have an internet connection. Those cases we would then also catch.

On the other hand it will make sense easier for us to maintain.